temporal/api/cloud/identity/v1/message.proto

```diff
@@ -60,6 +60,8 @@ message User {
     // The user specification
     UserSpec spec = 3;
     // The current state of the user
+    // Possible values: activating, activationfailed, active, updating, updatefailed, deleting, deletefailed, deleted, suspending, suspendfailed, suspended.
+    // For any failed state, reach out to Temporal Cloud support for remediation.
     string state = 4;
```

Comment:
We should consider making this an enum, but up to y'all.

Reply:
This is becoming an enum in a different PR.